hack/generate-docs.sh

```diff
@@ -27,12 +27,14 @@ source "${KUBE_ROOT}/hack/lib/init.sh"
 
 kube::golang::setup_env
 
-"${KUBE_ROOT}/hack/build-go.sh" \
-    cmd/gendocs \
-    cmd/genkubedocs \
-    cmd/genman \
-    cmd/genyaml \
-    federation/cmd/genfeddocs
+BINS=(
+	cmd/gendocs
+	cmd/genkubedocs
+	cmd/genman
+	cmd/genyaml
+	federation/cmd/genfeddocs
+)
+make -C "${KUBE_ROOT}" WHAT="${BINS[*]}"
```

Comment:
`[@]` expands all array elements as distinct "words" (tokens) whereas `[*]` does not.  We want them all to be one token in bash, so we can pass it to `make`